src/components/DriveView/Media.js

```diff
@@ -28,6 +28,9 @@ import { attachRelTime } from '../../analytics';
 import { fetchFiles, doUpload, fetchUploadUrls, fetchAthenaQueue, updateFiles } from '../../actions/files';
 import { clipsInit } from '../../actions/clips';
 
+const publicTooltip = 'Making a route public allows anyone with the route name or link to access it.';
+const preservedTooltip = 'Preserving a route will prevent it from being deleted. You can preserve up to 10 routes.';
```

Comment:
isn't this 100 routes?
```suggestion
const preservedTooltip = 'Preserving a route will prevent it from being deleted. You can preserve up to 100 routes.';
```

Reply:
It looks like it's 10 if you don't have prime, 100 if you do